### ISSUE TITLE
Python backend

### ISSUE THREAD

**Issue by vhermecz** [bobthecow/genghis#155]
Background: I've played with Genghis a while ago, and really liked its simple and clean UI, and source structure. As being more like a python person (than php or ruby), but being interested in ruby, I spent a few hours with understanding the ruby source, and porting it over to python.

I'd like to make this port available on Pypi, so this tool can reach more devs. So:
opt1 is that I could give you a pull request, and give support in extending the current rakefile to also create a python package.
opt2 - I'd use the client side app from your repo, and maintain/package/roll the python version myself. (guess i'd extract assets from genghis.rb if that's fine w you)

**Comment by bobthecow** [comment]
Yeah, that'd be great! Let's go with option 1, as that's consistent with how the Ruby implementation works as well.
